user-guide/Reference/DataMiner_Tools/Spectrum_Simulation_Tool.md

```diff
@@ -6,38 +6,14 @@ uid: Spectrum_Simulation_protocol
 
 To test DataMiner Spectrum Analysis, a spectrum simulation protocol is available. You can for example use it to verify the “set parameter” commands that SLSpectrum sends to the protocol, or to get a trace on the screen and test the various features of the spectrum user interface and spectrum monitoring without a real device.
 
-> You can download this protocol from [DataMiner Dojo](https://community.dataminer.services/download/spectrum-simulation-driver/).
-
 To use this protocol:
 
-1. Upload the protocol in the *Protocols & Templates* app in DataMiner Cube.
-
-1. Create an element that uses this protocol.
-
-The element has the following data pages:
-
-- *Spectrum Analyzer*: Displays the standard Spectrum Analysis user interface.
-
-- *General*: Contains general information, as well as the following parameters:
-
-  - *Track Parameter Sets*: Open the button next to this parameter to open a tracking window. This window allows you to trace which parameter sets were done. In the window, sets will only be logged if the *Track Sets* parameter is enabled. If *Track Status Sets* is also enabled, status sets (communication between SLSpectrum and the protocol, e.g. requesting a new trace) are also logged.
-
-  - *Trace pattern*: Allows you to select a specific trace pattern instead of the default pattern. The following patterns are available:
+1. Deploy the protocol from the [DataMiner Catalog](https://catalog.dataminer.services/details/6f33ec9f-e83d-49d5-8f85-87ad66eaa5c7).
 
```

Comment:
Double-checked this link, this is indeed very good. Having the link to the new protocol is what we wanted.